frontend/src/scenes/surveys/SurveyRepeatSchedule.tsx

```diff
@@ -29,9 +29,21 @@ function AlwaysScheduleBanner({
 }: {
     survey: Pick<Survey, 'type' | 'schedule' | 'conditions'>
 }): JSX.Element | null {
-    const { setSelectedSection } = useActions(surveyLogic)
+    const { setSelectedSection, setSurveyValue } = useActions(surveyLogic)
     const doesSurveyHaveWaitPeriod = (survey?.conditions?.seenSurveyWaitPeriodInDays ?? 0) > 0
 
+    const handleWaitPeriodClick = (): void => {
+        setSelectedSection(SurveyEditSection.DisplayConditions)
+        setSurveyValue('conditions', {
+            ...survey.conditions,
+            seenSurveyWaitPeriodInDays: 30,
```

Comment:
why do we add a `seenSurveyWaitPeriodInDays` here and not before?
the bug was just about the focus, unless you found a new bug?